tabpfn_client/tabpfn_classifier.py

```diff
@@ -1,69 +1,58 @@
 import logging
-from typing import Union
 from pathlib import Path
-import getpass
-import textwrap
+import shutil
 
 from sklearn.base import BaseEstimator, ClassifierMixin
 from sklearn.utils.validation import check_is_fitted
 
-from tabpfn import TabPFNClassifier as TabPFNClassifierLocal
-from tabpfn_client import tabpfn_service_client
-from tabpfn_client.tabpfn_service_client import TabPFNServiceClient
+from tabpfn import TabPFNClassifier as LocalTabPFNClassifier
+from tabpfn_client.remote_tabpfn_classifier import RemoteTabPFNClassifier
+from tabpfn_client.service_wrapper import UserAuthenticationClient, InferenceClient
+from tabpfn_client.client import ServiceClient
+from tabpfn_client.constants import CACHE_DIR
+from tabpfn_client.prompt_agent import PromptAgent
 
-CACHE_DIR = Path(__file__).parent.resolve() / ".tabpfn"
-TOKEN_FILE = CACHE_DIR / "config"
 
 logger = logging.getLogger(__name__)
 
 
 class TabPFNConfig:
     is_initialized = None
     use_server = None
+    user_auth_handler = None
+    inference_handler = None
 
 
 g_tabpfn_config = TabPFNConfig()
 
 
-def init(
-        use_server=True,
-):
+def init(use_server=True):
     global g_tabpfn_config
```

Comment:
given that we now already have something that this interacts with on the level above, it might be nice to move this function to be a member of tabpfn_client directly instead, what do you think?

Reply:
I'm not sure if I get you correctly here. But we can also briefly discuss this on Monday :)

Reply:
Will do this in this notion ticket [link](https://www.notion.so/Preliminary-Package-client-5fd3cf39290a45d3a94363cc4d31eefa?pvs=4)